CONTRIBUTING.md

```diff
@@ -371,6 +371,16 @@ Number Reservation for Documentation Use ([RFC 5398]). In particular:
 Both ranges have 16 total numbers each. The hexadecimal notation makes it more
 obvious where the range starts and stops.
 
+## Default Network Instance
+
+In OpenConfig [PR #599](https://github.com/openconfig/public/pull/599), it has
+been clarified that the name for the default network instance should be
+uppercase `"DEFAULT"`. Some legacy devices are still using lowercase
+`"default"`, so device tests should use the deviation
+`*deviations.DefaultNetworkInstance` which allows them to work on those legacy
+devices whlie they are being updated. Non-device unit tests may hard-code
+`"DEFAULT"`.
+
```

Comment:
Can this be moved to a new file such as DEVIATIONS.md?   The contributing guide can have a link added.  The  deviations document can include documentation of deviations as desired, but also importantly explain the intended use of deviations as temporary workarounds in order to demonstrate other portions of test and that deviations are treated like bugs, to be resolved in order to gain compliance.

Another option is to add a link to [deviations.go](https://github.com/openconfig/featureprofiles/blob/1bfcd6e1ef943d56c1d340e1a2405e02decf552c/internal/deviations/deviations.go) and expand on the documentation there.  But a markdown may look better, standout as documentation and enable easier to navigate links to specific deviations and their usage.

Reply:
As discussed, the reason this is in CONTRIBUTIONS.md is because this is being referenced by a pull-request check to explain why the string `"default"` is disallowed. None of the other deviations are checked in this manner.

Rather than adding a DEVIATIONS.md, I just added more detail to the deviations.go flag's help string.

To make sure tests are using the standardized `"DEFAULT"`, I will send a CL to Ondatra to add the constant to netutil, and follow up with another PR to redirect the deviation flag's default value to that constant.

Let me know if there is anything else you'd like me to add.